www/_posts/2017-09-08-ios-release.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+author:
+    name: Shazron Abdullah
+    url: https://twitter.com/shazron
+title:  "Cordova iOS 4.5.0"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce a minor version of `Cordova iOS 4.5.0` has been released!
+
+Three new features were added:
+1. [CB-12937](https://issues.apache.org/jira/browse/CB-12937) - Plugins can receive `handleOpenURLWithApplicationSourceAndAnnotation:` now (new selector, that sends the URL with additional metadata) 
+2. [CB-13164](https://issues.apache.org/jira/browse/CB-13164) - Integrated cordova-plugin-console to build in support for window.console. 
+3. [CB-10916](https://issues.apache.org/jira/browse/CB-10916) - Support [display name](/docs/en/dev/config_ref/index.html#name) for **iOS**
+
+Note that for [CB-13164](https://issues.apache.org/jira/browse/CB-13164) if you already included `cordova-plugin-console` in your project, you must **remove** that plugin if not your project will not build.
+
+If you ever needed to disable the built in console plugin, comment out or remove the `Console` `<feature>` tag in your platform specific `config.xml`, and/or call this right after the `deviceready` event:
```

Comment:
What is a "platform specific `config.xml`" - there is only one in my projects. Does this refer to `<platform name="ios">`?

Reply:
in platforms/platformName there is a platform specific config.xml for platformName

Reply:
Is that the only way of disabling it? should we make it configurable? Android has different log levels and can be configured via a preference in the config.xml

Reply:
Note that disabling the plugin will *not* get rid of the potential linker error conflict, it just disables the included plugin at runtime.

Reply:
Oh, didn't know about these config.xml files. These never get modified normally, do they?

(There is this "meme" in lots of StackOverflow, forum and blog posts that you should remove the console plugin before building a production build because "security" or "performance")

Reply:
They are build artifacts. Unfortunately this instruction is the best we can do with the current way its implemented. There is no way to remove "private" plugins to a platform. Since we are moving to npm in the way the CLI does things, the plugins could be local to the platform folder, and we could add them as dependencies instead which is installed by the CLI. That would make it less coupled like it is right now.